tests/std/tests/LWG3234_math_special_overloads/test.cpp

```diff
@@ -197,7 +197,7 @@ void test_comp_ellint_2() {
     static_assert(is_same_v<decltype(comp_ellint_2f(0)), float>);
     static_assert(is_same_v<decltype(comp_ellint_2l(0)), long double>);
     static_assert(is_same_v<decltype(comp_ellint_2(Ambiguous{})), Ambiguous>);
-    assert(comp_ellint_2(0) == acos(-1.0) / 2);
+    assert(expect_epsilons(comp_ellint_2(0), acos(-1.0) / 2, 2));

```

Comment:
For other reviewers: As discussed in https://github.com/boostorg/math/issues/909, this is an expected behavioral change due to re-implementation of the elliptical integrals to greatly increase performance for common cases at the expense of a bit of error for some special cases like π / 2. (No change requested.)

Reply:
Could we instead use the `isclose` function from #3077? I really dislike the use of epsilons here.